TinkerWorX.SharpCraft.Launcher/Program.cs

```diff
@@ -19,10 +19,29 @@ public static void Main(String[] args)
             // This fixes the current directory in case it is run from outside the launcher path.
             Environment.CurrentDirectory = Path.GetDirectoryName(typeof(SharpCraftApplication).Assembly.Location);
 
+            // Check for optional map specified to launch
+            string mapFile = ""; // Is there a loadfile arg specified?
+            for (var i = 0; i < args.Length; i++)
+            {
+                if (args[i].Contains(".w3x"))
```

Comment:
Indeed! I left out RoC .w3m files and replays. Those are now checked.

I changed the way this loop works as you suggested, finding the occurrence of the `-loadfile` argument, and checking the following argument. If the extension is .w3x, .w3m, or .w3g, we store that path. If not, drop a note in the console and just continue gracefully.